canvas_modules/common-canvas/src/common-properties/controls/textfield/textfield.jsx

```diff
@@ -37,10 +37,14 @@ class TextfieldControl extends React.Component {
 		this.charLimit = ControlUtils.getCharLimit(props.control, CHARACTER_LIMITS.TEXT_FIELD);
 		this.id = ControlUtils.getControlId(props.propertyId);
 		this.isList = false;
-		if (this.props.control.valueDef && this.props.control.valueDef.isList) {
-			this.isList = true;
+		if (this.props.control.valueDef) {
+			if (this.props.control.valueDef.isList) {
+				this.isList = true;
+			}
+			if (typeof this.props.control.valueDef.defaultValue !== "undefined") {
```

Comment:
Is there a reason to do this check?  It would just assign undefined to this.defaultValue anyway.

Reply:
You're right, fixed!